### ISSUE TITLE
custom-toastr css paths incorrect?

### ISSUE THREAD

**Issue by Spongsta** [zhukovka/react-toolbox-addons#8]
In this commit -> 10cd98a2365a7fb9718dbaf4edee779eddddcbfd
css imports were added.  Using webpack these referenced paths fail.  

Should they be `'../../../[css_file_path]';` instead of `'../../node_modules/[css_file_path]';`?

**Comment by zhukovka** [comment]
@Spongsta Yes, right, thank you for your report, @taraskrasiuk I'm still waiting for the fix from you
